docs/kb/api-version-3/emails.md

```diff
@@ -7,7 +7,7 @@ For a complete list of fields involved in user operations, see the [Email](../ob
 
 | **Operation** | **URL Format**                             | **Required Parameters** | **Description**  |
 | ------------- | ------------------------------------------ | ----------------------- | -----------------|
-| `read` | `/api/email/version/3/do/read/id/<email id>?...` | `user_key, api_key, email` | Returns the data for the email specified by `<id>`. `<id>` is the Pardot ID of the target email. |
+| `read` | `/api/email/version/3/do/read/id/<email id>?...` | `user_key, api_key, email_id` | Returns the data for the email specified by `<email_id>`. `<email_id>` is the Pardot ID of the target email. |
```

Comment:
This is picky but for the sake of uniformity, can you add an underscore to the email_id in /api/email/version/3/do/read/id/<email id> <--- is email id right now. Same for the other files.